core/src/commonMain/kotlin/logs/Logging.kt

```diff
@@ -41,7 +43,12 @@ public class Logging {
     }
 
     public fun interface DataProcessor {
-        public fun process(data: ByteArray): String
+        public fun process(
+            data: ByteArray,
+            serviceUuid: Uuid?,
+            characteristicUuid: Uuid?,
+            descriptorUuid: Uuid?,
```

Comment:
If this is likely to get more fields added in the future, would be nice to wrap these up in a `class` of some sort. Would still be breaking now, but would make it non-breaking to add fields in the future.

Reply:
Admittedly I'm taking a "move fast and break things" mindset with this, partly because it feels warranted while pre-1.x — but mostly because I want to ditch the data processor entirely in favor of #539. 🤷

Reply:
Added `@ObsoleteKableApi` to help warn consumers that the API will likely experience churn.